checker/tests/optional/OptionalRelyGuaranteeTest.java

```diff
@@ -0,0 +1,44 @@
+import java.util.List;
+import java.util.Optional;
+import org.checkerframework.checker.nonempty.qual.*;
+import org.checkerframework.checker.optional.qual.*;
+import org.checkerframework.dataflow.qual.Pure;
+
+// @skip-test: ignore until we have the correct checkers in-place
```

Comment:
Please expand on this comment so we know when to unskip the test.  Or, can it be unskipped now?